integration-cli/docker_api_stats_test.go

```diff
@@ -231,6 +231,9 @@ func (s *DockerSuite) TestApiStatsContainerNotFound(c *check.C) {
 func (s *DockerSuite) TestApiStatsContainerGetMemoryLimit(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 
+	if !isCgroupMounted("memory") {
```

Comment:
Perhaps this should be implemented as a testRequirement, see; https://github.com/docker/docker/blob/fd67c8b2031722dd9255996b84bbfb236ce45cfb/integration-cli/requirements.go#L25-L183

Then the line above can become (for example) `testRequires(c, DaemonIsLinux, MemoryCgroup)`